src/windows/index.rst

```diff
@@ -0,0 +1,92 @@
+.. deepstack-python documentation master file, created by
+   sphinx-quickstart on Sun Nov  8 22:05:48 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Using DeepStack with Windows 10 (CPU and GPU)
+=============================================
+
+DeepStack CPU and GPU Version is now available for the Windows operating System.
```

Comment:
To clarify, it is just windows 10?

Reply:
Yes it is just Windows 10. This is due to the ONNX components in the server.